### PR TITLE
add new fields

### PR DESCRIPTION
**Pull request by hiagolcm**
#### How to test it?

```
curl --request POST \
  --url 'https://hiago--exitocol.myvtex.com/_v/segment/graphql/v1?workspace=hiago' \
  --header 'Content-Type: application/json' \
  --data '{"query":"query recommendation($input: RecommendationInput) {\n  recommendation(input: $input) {\n    response {\n      recommendations {\n        base {\n          brand\n          brandId\n          cacheId\n          categories\n          description\n          items {\n            itemId\n            name\n            nameComplete\n            measurementUnit\n            unitMultiplier\n            ean\n            images {\n              imageId\n              imageLabel\n              imageTag\n              imageUrl\n              imageText\n            }\n            sellers {\n              sellerId\n              sellerName\n              sellerDefault\n              addToCartLink\n              commertialOffer {\n                AvailableQuantity\n                discountHighlights {\n                  name\n                }\n                teasers {\n                  name\n                  conditions {\n                    minimumQuantity\n                    parameters {\n                      name\n                      value\n                    }\n                  }\n                  effects {\n                    parameters {\n                      name\n                      value\n                    }\n                  }\n                }\n                Installments {\n                  InterestRate\n                  Name\n                  NumberOfInstallments\n                  TotalValuePlusInterestRate\n                  Value\n                }\n                ListPrice\n                Price\n                PriceWithoutDiscount\n                Tax\n                taxPercentage\n              }\n            }\n            referenceId {\n              Key\n              Value\n            }\n            variations {\n              name\n              values\n            }\n          }\n          link\n          linkText\n          priceRange {\n            listPrice {\n              highPrice\n              lowPrice\n            }\n            sellingPrice {\n              highPrice\n              lowPrice\n            }\n          }\n          productId\n          productName\n          productReference\n          specificationGroups {\n            name\n            originalName\n            specifications {\n              name\n              originalName\n              values\n            }\n          }\n        }\n        recommended {\n          brand\n          brandId\n          cacheId\n          categories\n          description\n          items {\n            itemId\n            name\n            nameComplete\n            measurementUnit\n            unitMultiplier\n            ean\n            images {\n              imageId\n              imageLabel\n              imageTag\n              imageUrl\n              imageText\n            }\n            sellers {\n              sellerId\n              sellerName\n              sellerDefault\n              addToCartLink\n              commertialOffer {\n                AvailableQuantity\n                discountHighlights {\n                  name\n                }\n                teasers {\n                  name\n                  conditions {\n                    minimumQuantity\n                    parameters {\n                      name\n                      value\n                    }\n                  }\n                  effects {\n                    parameters {\n                      name\n                      value\n                    }\n                  }\n                }\n                Installments {\n                  InterestRate\n                  Name\n                  NumberOfInstallments\n                  TotalValuePlusInterestRate\n                  Value\n                }\n                ListPrice\n                Price\n                PriceWithoutDiscount\n                Tax\n                taxPercentage\n              }\n            }\n            referenceId {\n              Key\n              Value\n            }\n            variations {\n              name\n              values\n            }\n          }\n          link\n          linkText\n          priceRange {\n            listPrice {\n              highPrice\n              lowPrice\n            }\n            sellingPrice {\n              highPrice\n              lowPrice\n            }\n          }\n          productId\n          productName\n          productReference\n          specificationGroups {\n            name\n            originalName\n            specifications {\n              name\n              originalName\n              values\n            }\n          }\n          properties {\n            name\n            values\n          }\n        }\n      }\n    }\n  }\n}\n","variables":{"input":{"sessionId":"1981879a36f2eb2c0899ac18f24e08982718b503","strategy":"MOST_POPULAR","input":{"type":{"primary":"STORE"}},"recommendation":{"count":{"minimum":5,"recommendations":20}}}},"operationName":"recommendation"}'
```

### CODE REVIEW

**Review by ullayne02 — APPROVED** [review]
:hiagod:

File: react/graphql/recommendation.gql [hunk 1]
```diff
@@ -29,6 +29,25 @@ query recommendation($input: RecommendationInput) {
               addToCartLink
               commertialOffer {
                 AvailableQuantity
+                discountHighlights {
```
Comment (ullayne02) [inline react/graphql/recommendation.gql:32]: These attributes should also be in the product search query?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- `teasers`, `discountHighligh` and properties to the `recommendation` query.
```
Comment (chrsmutti) [inline CHANGELOG.md:11]:
```suggestion
- `teasers`, `discountHighlights` and properties to the `recommendation` query.
```

**Review by chrsmutti — APPROVED** [review]
LGTM

**Pull request merged**